snow/engine/snowman/transitive_test.go

```diff
@@ -3069,41 +3013,37 @@ func TestEngineBubbleVotesThroughInvalidChain(t *testing.T) {
 			*reqVdr = inVdr
 			return
 		default:
-			t.Fatal("Unexpectedly sent a Get request for unknown block")
+			require.FailNow("Unexpectedly sent a Get request for unknown block")
 		}
 	}
 
+	// this VM expects to receive "Chits" in response to the "PullQuery"
+	// not to send out a "PullQuery"
 	sender.SendPullQueryF = func(_ context.Context, _ set.Set[ids.NodeID], _ uint32, blkID ids.ID) {
```

Comment:
Yikes this is jank - we just shouldn't set `SendPullQueryF` here right? if the var is `nil` then it will already error

Reply:
It will actually error, not panic: https://github.com/ava-labs/avalanchego/blob/master/snow/engine/common/test_sender.go#L259